owncloudApp/src/main/java/com/owncloud/android/presentation/transfers/TransferListComposeFragment.kt

```diff
@@ -0,0 +1,532 @@
+                        modifier = Modifier
+                            .layoutId("uploadListGroupButtonClear")
+                            .padding(start = 13.dp),
+                        onClick = {
+                            if (status == TransferStatus.TRANSFER_FAILED) {
+                                transfersViewModel.clearFailedTransfers()
+                            } else {
+                                transfersViewModel.clearSuccessfulTransfers()
+                            }
+                        },
+                        shape = RoundedCornerShape(1.dp),
+                        colors = ButtonDefaults.buttonColors(backgroundColor = colorResource(id = R.color.primary_button_background_color))
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.action_upload_clear).uppercase(),
+                            color = colorResource(id = R.color.white),
+                        )
+                    }
+                }
+                if (status == TransferStatus.TRANSFER_FAILED) {
+                    Button(
+                        modifier = Modifier
+                            .layoutId("uploadListGroupButtonRetry")
+                            .padding(start = 8.dp),
+                        onClick = { transfersViewModel.retryFailedTransfers() },
+                        shape = RoundedCornerShape(1.dp),
+                        colors = ButtonDefaults.buttonColors(backgroundColor = colorResource(id = R.color.color_accent))
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.action_upload_retry).uppercase(),
+                            color = colorResource(id = R.color.white),
+                        )
+                    }
+                }
+
+            }
+        }
+        if (status == TransferStatus.TRANSFER_SUCCEEDED || status == TransferStatus.TRANSFER_FAILED){
+            Divider(
+                modifier = Modifier
+                    .fillMaxWidth(),
+                color = colorResource(id = R.color.filelist_icon_background),
+                thickness = 1.dp
+            )
+        }
+
+    }
+
+    private fun headerTitleStringRes(status: TransferStatus): Int {
+        return when (status) {
+            TransferStatus.TRANSFER_IN_PROGRESS -> R.string.uploads_view_group_current_uploads
+            TransferStatus.TRANSFER_FAILED -> R.string.uploads_view_group_failed_uploads
+            TransferStatus.TRANSFER_SUCCEEDED -> R.string.uploads_view_group_finished_uploads
+            TransferStatus.TRANSFER_QUEUED -> R.string.uploads_view_group_queued_uploads
+        }
+    }
+
+    private fun checkAccount(transfer: OCTransfer): String {
+        return try {
+            val account = AccountUtils.getOwnCloudAccountByName(requireContext(), transfer.accountName)
+            val oca = OwnCloudAccount(account, requireContext())
```

Comment:
`oca` is not a suitable name for a variable, better setting a larger name and know what it is containing 😄

Reply:
This is the name that is in legacy version. Now, I've replaced this name by `ownCloudAccount`